src/doc/en/developer/walkthrough.rst

```diff
@@ -234,40 +212,40 @@ changes::
 
 To dig deeper into what was changed in the files you can use::
 
-    [user@localhost sage]$ git diff some_file.py
+    [alice@localhost sage]$ git diff some_file.py
 
 to show you the differences.
 
 
-
 .. _section-walkthrough-make:
 
 Rebuilding Sage
 ===============
 
-Once you have made any changes you of course want to build Sage and
```

Comment:
Add here: "All changes to Python files take effect immediately after restarting Sage (unless you have used `./configure --disable-editable`.)

Reply:
Yes. I had added a sentence for the same effect. But I replaced it with a longer note.